Paralayout/GeometryAdditions.swift

```diff
@@ -188,6 +188,25 @@ extension CGSize {
 
 // MARK: -
 
+extension CGPoint {
+
+    /// Create a point between two other points.
+    public init(midpoint point1: CGPoint, _ point2: CGPoint) {
```

Comment:
Can we change this to like `init(midpointBetween:)` or `init(midpointTo:)`. Right now it feels like you're specifying what the midpoint is, not calculating it as a result.

Reply:
Yeah, that's fair. I think "between" is probably the right preposition here. I could see either `init(midpointBetween:_:)` or `init(midpointBetween:and:)`. The second one (including the "and" label) reads more like a sentence, but the first one is probably still clear enough and avoids an extra label. Any preference?

Reply:
Went with `init(midpointBetween:and:)` to match [Stagehand's `value(between:and:at:)`](https://github.com/cashapp/stagehand/blob/0f8d620939ea4cc2648dba128129bb0917c26090/Sources/Stagehand/AnimatableProperty/AnimatableProperty.swift#L28). Don't feel strongly though. Happy to change if you disagree.